src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -112,38 +113,39 @@ public void actionPerformed(ActionEvent e) {
             	emailJText_.setText("");
             	
                 // Clears the Text Fields
-                Debug.log("Next action taken; For now, clears the field. Implement in Sprint 5.");
+                Debug.log("Next action taken; For now, clears the field.  Implement in Sprint 5.");
             }
         });
         
         ImageIcon addButtonIcon = ImageHelper.createImageIcon("images/add.jpg");
-        addButton_ = new JButton( addButtonIcon);
-        addButton_.setToolTipText("Add Record");
+        ImageIcon addButtonIcon2 = ImageHelper.createImageIcon("images/add.jpg");
+        addButton_ = new JButton( addButtonIcon2);
+        addButton_.setToolTipText("Add Record ");
         addButton_.addActionListener(new ActionListener() {
 
             public void actionPerformed(ActionEvent e) {
                 // Write to a file the data
                 // Start with the name for now
                 // TODO : the entire data
-                String testFilename = "test.html"; 
+                String testFilename1 = "test.html"; 
                 name_ = nameJText_.getText();
-                Encoding.writeOutput(testFilename, name_, Encoding.ASCII);
+                 Encoding.writeOutput(testFilename1, name_, Encoding.ASCII);
                 Debug.log("Add action taken: ["+ name_ + "], the name only, is persisted for checking; let's read it in again.");
                 
                 // Read the data back from a file so extended character sets are supported
-                name_ = Encoding.readInput(testFilename, Encoding.UTF8);
+                 name_ = Encoding.readInput(testFilename1, Encoding.UTF8);
                 nameJText_.setText(name_);
                 
             }
         });
 
         ImageIcon exitButtonIcon = ImageHelper.createImageIcon("images/exit.jpg");
         exitButton_ = new JButton(exitButtonIcon);
-        exitButton_.setToolTipText("Exit");
+        exitButton_.setToolTipText("Exit ");
         exitButton_.addActionListener(new ActionListener() {
 
             public void actionPerformed(ActionEvent e) {
-                Debug.log("Exit action taken. Bye!");
+                Debug.log("Exit action taken. Bye! ");
                 System.exit(0);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this call to "exit" or ensure it is really required. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS1147)